### ISSUE TITLE
Investigate issues with datasets 

### ISSUE THREAD

**Issue by ESapenaVentura** [ebi-ait/hca-ebi-wrangler-central#418]
For more context: https://humancellatlas.slack.com/archives/C01360XN04S/p1630422384014400

Raised by Valerie Reeves on 31/8/2021 

1. **snRNA-seq_for_human_retina**, **CrohnsTherapyResistance**, and **TxDynEndotheilialCellsFromPluriStem** all still have issues with the data that we identified in previous releases, but the issues haven't been fixed yet.
1. **KidneyOrganoidsBioprinting** has a similar issue as one of the rows in the Crohns project. The input_name seems to have multiple names in it.
1. Six rows of the **Stress-inducedRNA–chromatininteractionspromoteendo** `participant_lane_set` table seem to be missing "organ info" causing the following error in GetTerraMetadata:

   ```
   Failed to evaluate '__project_stratum_string' (reason 1 of 1): Evaluating select_first([project_stratum_string, ((((((("project=" + __project_id) + ";library=") + library[0]) + ";species=") + species[0]) + ";organ=") + organ[0])]) failed: Bad array access organ[0]: Array size 0 does not have an index value '0'
   ```

**Comment by ESapenaVentura** [comment]
Point 1: 
- Crohntherapy issues: https://github.com/ebi-ait/hca-ebi-wrangler-central/issues/240
- TxDynEndothelialCellsFromPluriStem (UCSC dataset): Labelled `file_index` field incorrectly
- snRNA-seq_for_human_retina: More than 1 set of files per bundle. **DCP1 DATASET**

Point 2:
- KidneyOrganoidsBioprinting: More than 1 set of files per bundle, no lane index

Point 3:
- Stress-inducedRNA–chromatininteractionspromoteendo: This doesn't exist in ingest?!? https://data.humancellatlas.org/explore/projects/87d52a86-bdc7-440c-b84d-170f7dc346d9 is it a stanford dataset?

**Comment by ami-day** [comment]
Point 2 KidneyOrganoidsBioprinting: I updated the spreadsheet with lane indices, but I think the problem is having multiple cell suspension ids for each single run / pair of fastq files. This was intended due to the sequencing experiment being run on pooled sample replicates, though i am not sure if this is an appropriate structure for pipelines to process currently

**Comment by ami-day** [comment]
I have fixed and resubmitted the metadata for 1. CrohnsTherapyResistance

**Comment by ami-day** [comment]
Added the internally blocked label because updates to this dataset: KidneyOrganoidsBioprinting are blocked by the inability to pipelines to take multiplexed sequence data. Enrique, Wei and I are meeting this afternoon to discuss way forward.

**Comment by amnonkhen** [comment]
@ami-day @Wkt8 @ESapenaVentura is this a single ticket? should it be split into 3 tickets (one for each dataset)?

**Comment by jahilton** [comment]
>Six rows of the Stress-inducedRNA–chromatininteractionspromoteendo participant_lane_set table seem to be missing "organ info" causing the following error in GetTerraMetadata:

Confirmed that my team submitted that dataset. Can someone clarify the issue here? I am unfamiliar with the "table" being mentioned here & what "organ info" is.

**Comment by ami-day** [comment]
> > Six rows of the Stress-inducedRNA–chromatininteractionspromoteendo participant_lane_set table seem to be missing "organ info" causing the following error in GetTerraMetadata:
> 
> Confirmed that my team submitted that dataset. Can someone clarify the issue here? I am unfamiliar with the "table" being mentioned here & what "organ info" is.

Hi @jahilton I am not sure what this means either, as we are not able to validate and submit datasets without organ metadata via dcp ingest. This description originally came from Valerie Reeves, are you/your team able to get in touch with Valerie about this?

**Comment by ami-day** [comment]
> @ami-day @Wkt8 @ESapenaVentura is this a single ticket? should it be split into 3 tickets (one for each dataset)?

I think for now it is useful to have this 1 ticket with all the info., if there are any longer term issues that we can't fix now (including submitting multiplexed data, we're meeting with pipelines about this on Weds) then I will create new separate tickets.

**Comment by ami-day** [comment]
Creating a new separate ticket for the snRNA-seq_for_human_retina dataset. The submission in ingest prod. is here: https://contribute.data.humancellatlas.org/submissions/detail?uuid=7d515499-b5b6-4083-95cd-6a393344865e&project=9c20a245-f2c0-43ae-82c9-2232ec6b594f and it was wrangled by Zina. However it is very old, a DCP1 dataset. We cannot currently update DCP1 datasets. Hence, creating a separate ticket.

**Comment by Wkt8** [comment]
snRNA-seq_for_human_retina has a separate ticket
CrohnsTherapyResistance is complete 
TxDynEndotheilialCellsFromPluriStem is a UCSC dataset (wei to notify)

KidneyOrganoidsBioprinting is complete

Stress-inducedRNA–chromatininteractionspromoteendo is submitted directly to Terra by Stanford and they are using an out-of-date version of the schema (2019) which is missing model_organs.
